internal/github/github.go

```diff
@@ -87,6 +88,11 @@ func (r *RealGitHub) ClosePullRequest(output io.Writer, workingDir string, branc
 	return execInstance.Execute(output, workingDir, "gh", "pr", "close", fmt.Sprint(pr.Number))
 }
 
+func (r *RealGitHub) GetDefaultBranchName(output io.Writer, workingDir string, fullRepoName string) (string, error) {
+	defaultBranch, err := execInstance.ExecuteAndCapture(output, workingDir, "gh", "repo", "view", fullRepoName, "--json", "defaultBranchRef", "--jq", ".defaultBranchRef.name")
+	return strings.Trim(defaultBranch, "\n"), err
```

Comment:
the command returns the branch name with a \n appended. Tried a few other options, e.g. 
`gh repo view bd/reviewstruck --json defaultBranchRef --template {{.defaultBranchRef.name}}`
`gh api repos/bd/reviewstruck --template {{.default_branch}}`
but the same happens.